tuf/developer_tool.py

```diff
@@ -75,7 +74,8 @@
     import_rsa_privatekey_from_file)
 
 from securesystemslib.keys import (
-    format_keyval_to_metadata)
+    format_keyval_to_metadata,
+    format_metadata_to_key)
```

Comment:
Curious why you chose this pattern here, importing specific functions, but not in updater?

Reply:
oh just because developer_tool was already using the "from x import y" pattern for keys (just not consistently). It's likely that other places would benefit from doing that but in those cases I would have had more new local name conflicts to worry about (when I import the module, I can just check the module name is not used)